CONTRIBUTING.md

```diff
@@ -30,6 +30,12 @@ before you import the SDK
    git pull
    ```
 
+1. Generate Documentation
+
+   ```
+   npm run docs
```

Comment:
Can we change the `release` script to the below instead of having two commands to run.

```
npm run docs && oneblink release
```